### PR TITLE
Add "Workshop Suggestions" board

### PR DESCRIPTION
**Pull request by thekakamora**

### CODE REVIEW

**Comment by jfmyers** [comment]
Hey @thekakamora this looks great. Thanks for creating the PR.

One question, did you first generate the board in Canny?

File: frontend/lib/stores/BountyProgramsStore.ts [hunk 1]
```diff
@@ -37,6 +37,11 @@ const programs = [
     slug: "uniswap",
     bountyProgramID: "f6d8778a-6f1d-b643-5661-b1ca97f391ed",
   },
+  {
+    name: "Workshop Suggestions",
+    slug: "workshop-suggestions",
+    bountyProgramID: "f4d6f311-3853-2b7b-a492-07e9608efe19",
```
Comment (jfmyers) [inline frontend/lib/stores/BountyProgramsStore.ts:43]: Is this `bountyProgramID` a board id from Canny?

Comment (thekakamora) [reply frontend/lib/stores/BountyProgramsStore.ts:43]: Yes, I setup a board on our Canny (https://metricsdao.canny.io/admin/settings/boards/workshop-suggestions/general) and got its identifier.

**Comment by thekakamora** [comment]
Closing the pull request as we decided to move to discourse to manage workshops.

**Pull request closed without merging**